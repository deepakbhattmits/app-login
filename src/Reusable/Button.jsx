import React from "react";

const Button = props => {
  const { id, type, name, className, onClick, title, children } = props
  return (
    <button
      id={id ? id : ""}
      name={name ? name : ''}
      type={type ? type : ""}
      className={className ? className : ""}
      onClick={onClick}
      title={title ? title : ""}
    >
      {children}
    </button>
  );
};
export default Button;
