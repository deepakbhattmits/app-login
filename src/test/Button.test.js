import React from 'react'
import { shallow, configure } from 'enzyme'
import Button from '../Reusable/Button'
import EnzymeAdapter from 'enzyme-adapter-react-16';
configure({ adapter: new EnzymeAdapter() });

test('render without crash', () => {
    const wrapper = shallow(<Button />)
    expect(wrapper).toMatchSnapshot();
});