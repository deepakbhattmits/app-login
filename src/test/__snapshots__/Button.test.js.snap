// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render without crash 1`] = `ShallowWrapper {}`;
